classes/Pods/Data.php

```diff
@@ -978,7 +978,20 @@ public function build( $params ) {
 		}
 
 		if ( ! empty( $params->orderby ) ) {
+			if ( 'post_type' == $pod[ 'type' ] && 'meta' == $pod[ 'storage' ] && is_array( $params->orderby ) ) {
+				foreach ( $params->orderby as $i => $orderby ) {
+					if ( strpos( $orderby, '.meta_value_num' ) ) {
+						$params->orderby[ $i ] = 'CAST(' . str_replace( '.meta_value_num', '.meta_value', $orderby ) . ' AS DECIMAL)';
+					} elseif ( strpos( $orderby, '.meta_value_date' ) ) {
+						$params->orderby[ $i ] = 'CAST(' . str_replace( '.meta_value_date', '.meta_value', $orderby  ) . ' AS DATE)';
+					}
+
+				}
+
+			}
+			
 			$params->orderby = (array) $params->orderby;
```

Comment:
Right, it was good in 2.x, but in this case $params can't be assumed it's an array yet.